try:
    num = 2

    while num < 5:
        print (num)
        
except Exception as error:
    print(error)
